docs/content/operations/api-reference.md

```diff
@@ -230,6 +230,18 @@ Enables all segments of datasource which are not overshadowed by others.
 
 Enables a segment of a datasource.
 
+* `/druid/coordinator/v1/datasources/{dataSourceName}/markUsed`
+
+Enables segments of a datasource by the interval or by an array of segment id's passed in the request payload. 
```

Comment:
Please add the below two things:

- The segments would be enabled when this API is called, but then can be disabled by the coordinator if no loadRules match or any dropRule matches.
- If a kill task was running to kill same segments at the same time, the behavior is not defined. Some segments might be killed and others might be enabled. Please add a warning to not use the kill task and this API at the same time.